pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "describealign"
+authors = [{ name = "Julian Brown", email = "julbean@proton.me" }]
+description = "Combines videos with matching audio files (e.g. audio descriptions)"
+readme = "README.md"
+requires-python = ">=3.8"
+dependencies = [
+  "ffmpeg_python >= 0.2.0",
+  "static-ffmpeg >= 2.5",
+  "matplotlib >= 3.5.0",
+  "numpy >= 1.21.4",
+  "python_speech_features >= 0.6",
+  "scipy >= 1.10.1",
+  "pytsmod >= 0.3.7",
+  "PySimpleGUIWx==0.17.2 ; platform_system == 'Windows'",
+  "PySimpleGUIQt>=0.35.0 ; platform_system != 'Windows'",

```

Comment:
We'll need to change this to "==" to keep it from using the new paid version

Reply:
Ah yes, I meant to switch this to read the requirements.txt so we only have one source of truth, for simplicity

Reply:
Fixed